packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/cancel-deal.tsx

```diff
@@ -75,17 +75,17 @@ const CancelDeal = observer(() => {
                     icon='info'
                     id='dt_cancellation-checkbox__tooltip'
                     is_bubble_hover_enabled
-                    message={localize(
```

Comment:
Does it make sense to replace all `localize()`? For e.g. on lines 40, 57, 58. But in that case there is a need to change types of expected props

Reply:
@kate-deriv sure, i can find some more to replace which don't require types change, e.g. Checkbox label